core/src/main/java/com/datastax/oss/driver/internal/core/config/cloud/CloudConfigFactory.java

```diff
@@ -58,6 +59,8 @@
 @ThreadSafe
 public class CloudConfigFactory {
   private static final Logger LOG = LoggerFactory.getLogger(CloudConfigFactory.class);
+  private static final int METADATA_RETRY_MAX_ATTEMPTS = 4;
+  private static final int METADATA_RETRY_INITIAL_DELAY_MS = 250;
```

Comment:
Note: per an earlier conversation @SiyaoIsHiding and I decided to leave these as constants for now.  We can move them to configurable values if an explicit need prevents itself (or perhaps as a future enhancement).